azext_iot/product/params.py

```diff
@@ -0,0 +1,149 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+"""
+CLI parameter definitions.
+"""
+
+from azure.cli.core.commands.parameters import get_three_state_flag, get_enum_type
+from azext_iot.product.shared import AuthType, BadgeType, TaskType
+
+
+def load_product_params(self, _):
+    with self.argument_context('iot product') as c:
+        c.argument(
+            'product_name',
+            options_list=['--product-name'],
+            help='Product name to display in catalog',
+            arg_group='IoT Device Certification'
+        )
+        c.argument(
+            'test_id',
+            options_list=['--test-id'],
+            help='The generated Id for the device certification test',
+            arg_group='IoT Device Certification'
+        )
+        c.argument(
+            'badge_type',
+            options_list=['--badge-type', '--bt'],
+            help='The type of certification badge',
+            arg_group='IoT Device Certification',
```

Comment:
cool, glad you using `arg_group`

Reply:
Wait until you see the upcoming `az iot product test create`. If you aren't using the `--configuration-file` you have to specify attestation and device definition args. To keep from presenting a wall of args, they are grouped together.